core/domain/user_id_migration.py

```diff
@@ -539,11 +715,8 @@ def map(model):
                 yield ('FAILURE - %s' % model_class.__name__, model.id)
         elif (model_class.get_user_id_migration_policy() ==
               base_models.USER_ID_MIGRATION_POLICY.COPY_AND_UPDATE_ONE_FIELD):
-            user_id = model.user_id
-            if user_id is None:
-                yield ('SUCCESS_NONE - %s' % model_class.__name__, model.id)
-            elif (BaseModelsUserIdsHaveUserSettingsVerificationJob
-                  ._check_id_and_user_id_exist(model.id, user_id)):
+            if (BaseModelsUserIdsHaveUserSettingsVerificationJob
```

Comment:
It can't any longer happen that the `user_id` would be `None` in this branch of code, all the models with `COPY_AND_UPDATE_ONE_FIELD` have the field containing user ID always set to something. (This was verified in the previous runs of the migration)

Reply:
You might still consider keeping it for safety. If someone adds a new model type before the actual migration and there's an error in the field, this would at least catch it.

Reply:
Done, readded.